File: Client/Models/Address.cs
using System;

namespace Client2.Models
{
    public class AddressT
    {
        public int id {get; set;}
        public string city {get; set;}
        public string street {get; set;}
        public string zip {get; set;}

        public string toString => $"{city},{street}, {zip}";
    }
}

File: Server/src/main/java/uae/ensate/rentudiant/model/Address.java
package uae.ensate.rentudiant.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Set;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "addresses")
public class Address {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    private String city;

    private String street;

    private String zip;

    public Address(String city, String street, String zip) {
        this.city = city;
        this.street = street;
        this.zip = zip;
    }

}
